examples/views/button_roles.py

```diff
@@ -26,11 +23,12 @@ def __init__(self, role: discord.Role):
                          style=discord.enums.ButtonStyle.primary, custom_id=str(role.id))
 
     async def callback(self, interaction: discord.Interaction):
-        """This function will be called any time a user clicks on this button
+        """
+        This function will be called any time a user clicks on this button
```

Comment:
```suggestion
        """This function will be called any time a user clicks on this button
```
consistency with project code style